mbofui/app/js/controllers.js

```diff
@@ -95,31 +95,29 @@ ocellus.controller('mapController', ['$scope', '$rootScope','$filter', '$timeout
     var endTime = moment($('#endTime').val()).format($rootScope.time_format);
     var postingTime = moment().format($rootScope.time_format);
     var data = {
-      "eventText": $scope.newEventText,
-      "startTime": startTime,
-      "endTime": endTime,
-      "latitude": coords[0],
-      "longitude": coords[1],
-      "altitudeMeters": 266.75274658203125, //this is being removed but the database still expects it
-      "postingTime": postingTime
+      'eventText': $scope.newEventText,
+      'startTime': startTime,
+      'endTime': endTime,
+      'latitude': coords[0],
+      'category':$scope.selected_category.key,
+      'longitude': coords[1],
+      'altitudeMeters': 266.75274658203125, //this is being removed but the database still expects it
+      'postingTime': postingTime
     };
     // use function in utils.js to see if the data validates
     var validationFailures = validate(data);
     //if there were no validation failures, post it
     // and construct a new marker to add to map
     if(!validationFailures.length) {
       Bof.PostBof(url, data).then(function(result) {
+        var category = Bof.resolveCategory(result.data.category);
         var newMarker = {
           lat: result.data.latitude,
           lng: result.data.longitude,
-          category: 'cat1',
-          event: result.data.eventText,
+          category: category.key,
+          message: '<strong>' + category.label + '</strong><br>' + result.data.eventText,
```

Comment:
You're replacing `event` with `message` here.  Isn't that backwards?  It was just changed from `message` to `event` recently.